src/modules/fw_pos_control/FixedwingPositionControl.hpp

```diff
@@ -482,6 +488,7 @@ class FixedwingPositionControl final : public ModuleBase<FixedwingPositionContro
 	 */
 	float get_terrain_altitude_takeoff(float takeoff_alt);
 
+	float getMaximumClimbRate();
```

Comment:
can you write a quick brief on usage? (that it is a function of density + weight)